build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.20"
+    application
+
+    id("org.jlleitschuh.gradle.ktlint") version "10.2.1"
+    id("maven-publish")
+}
+
+group = "com.flagsmith"
+version = "1.0.0"
```

Comment:
I'm not sure of the conventions with regards to Kotlin / Java packages. Should the Kotlin client follow the version numbers of the Java client, or is it fine to have them totally different?

Reply:
```suggestion
version = "5.0.0"
```